book/06-github/sections/1-setting-up-account.asc

```diff
@@ -1,95 +1,95 @@
+Ang panghuling address ay hindi pa napatunayan, ibig sabihin na hindi mo maaaring gawin itong pangunahing address.
+Kung nakikita ng GitHub ang alin man sa mga ito sa mga mensahe ng commit sa alinmang repositoryo sa site, ito ay maiuugnay na ngayon sa iyong user.
 
-==== Two Factor Authentication
+==== Dalawang Kadahilanan ng Pagpapatunay
 
-Finally, for extra security, you should definitely set up Two-factor Authentication or ``2FA''.
-Two-factor Authentication is an authentication mechanism that is becoming more and more popular recently to mitigate the risk of your account being compromised if your password is stolen somehow.
-Turning it on will make GitHub ask you for two different methods of authentication, so that if one of them is compromised, an attacker will not be able to access your account.
+Sa wakas, para sa dagdag na seguridad, kailangan mong mag-setup ng Dalawang-Kadahilanan ng Pagpapatunay o ``2FA''.
+Ang Dalawang-Kadahilanan ng Pagpapatunay ay isang mekanismo ng pagpapatunay na nagiging mas at mas tanyag kumakailan lamang upang mapigilan ang panganib na makompromiso ang iyong account kung ang iyong password ay nanakaw.
+Ang pag-on nito ay maging dahilan ng paghihiling sa iyo ng GitHub para sa dalawang iba't-ibang mga paraan ng pagpapatunay, upang kapag isa sa mga ito ay nakompromiso, hindi maa-access ng attacker ang iyong account.
 
-You can find the Two-factor Authentication setup under the Security tab of your Account settings.
+Maaari mong mahanap ang setup ng Dalawang-Kadahilanan ng Pagpapatunay sa ilalim ng tab ng Seguridad sa settings ng iyong Account.
 
-.2FA in the Security Tab
+.2FA sa Tab ng Seguridad
 image::images/2fa-1.png[2FA in the Security Tab]
 
-If you click on the ``Set up two-factor authentication'' button, it will take you to a configuration page where you can choose to use a phone app to generate your secondary code (a ``time based one-time password''), or you can have GitHub send you a code via SMS each time you need to log in.
+Kapag i-click mo ang pindutan na ``Mag-setup ng dalawang-kadahilanan ng pagpapatunay'', dadalhin ka nito sa pahina ng pagsasaayos kung saan maaari kang mamili na gumamit ng isang app sa phone upang makabuo ng pangalawangcode (``isang-beses na password'' na nabatay sa oras), o maaari mong hayaan ang GitHub na magpadala sa iyo ng isang code sa pamamagitan ng SMS sa bawat oras na kailangan mong mag-login.
 
-After you choose which method you prefer and follow the instructions for setting up 2FA, your account will then be a little more secure and you will have to provide a code in addition to your password whenever you log into GitHub.
+Pagkatapos mong pumili kung anong paraan ang mas gusto mo at sundin ang mga instruksiyon sa pag-setup ng 2FA, ang iyong account ay magiging mas ligtas nang kaunti at kinakailangan mong magbigay ng isang code bilang karagdagan sa iyong password sa tuwing ikaw ay mag-login sa GitHub.
```

Comment:
i think mga direksiyon or mga pagtuturo  would fit for instructions  instead of instruksiyon

Reply:
hi @severinolorillajr, which will i use?